OpenEdXMobile/res/values/strings.xml

```diff
@@ -467,11 +467,11 @@
     <!-- Prompt leading user to sign in screen -->
     <string name="already_have_an_account">Already have an account? Sign in</string>
     <!-- Prompt leading user to create account screen -->
-    <string name="sign_up_and_learn">Sign up and start learning</string>
-    <!-- In Sign up screen -->
-    <string name="sign_up_with">Sign up with</string>
-    <!-- In Sign up screen -->
-    <string name="or_sign_up_with_email">or sign up with email</string>
+    <string name="register_and_learn">Register and start learning</string>
```

Comment:
@mdinino You need to refactor the string with the IDE. If you only change the text here, the references to this string won't be updated.